client/src/tutorials/ExpenseTracker/files/lib.rs

```diff
@@ -0,0 +1,138 @@
+use anchor_lang::prelude::*;
+use anchor_lang::solana_program::{self,system_program, sysvar::rent::Rent,};
+
+// declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
+declare_id!("Dt1Cu5jz7UMzLFgXPNyYcP8jCcySYgLW5QXpubjKvvm3");
+
+
+
+#[program]
+pub mod etracker {
+
+    use super::*;
+
+    pub fn initialize_expense(ctx: Context<InitializeExpense>,id : String,mname : String,amount : u64) -> Result<()> {
+        
+        let expense_account = &mut ctx.accounts.expense_account;
+        let integer_id = id.parse::<u64>().unwrap();
+        let string_id = integer_id.to_string();
+
+        require_eq!(string_id, id);
+
+        expense_account.id = integer_id;
+        expense_account.mname = mname;
+        expense_account.amount = amount;
+        expense_account.owner = *ctx.accounts.authority.key;
+        expense_account.bump = *ctx.bumps.get("expense_account").unwrap();
```

Comment:
Saving bumps is not necessary, using `#[account(bump, ...)]` will get the same result.